BrainPortal/app/views/tools/show.html.erb

```diff
@@ -0,0 +1,139 @@
+
```

Comment:
Please de-tab this file and remove trailing spaces, as per

https://github.com/aces/cbrain/wiki/Programming-Style